common.mk

```diff
@@ -284,6 +284,16 @@ append-var_ = echo '$(1)=$(3)'$($(1))'$(3)' >>$(2);
 append-var = $(call append-var_,$(1),$(2),$(if $(call y-or-n,$($(1))),,$(double-quote)))
 append-br2-vars = $(foreach var,$(filter BR2_%,$(.VARIABLES)),$(call append-var,$(var),$(1)))
 
+# When building with support for measured boot and an fTPM, there is a circular
```

Comment:
Defining ibmtss as a buildroot package should take care of this. Or am I missing something?

Reply:
Hi,

I based this patch on an older patch a colleague sent me some time ago and I didn't think of that, actually. Let me give it a try to see how it works.

Thanks for the suggestion @jenswi-linaro

Reply:
Hi @jenswi-linaro,

Following your suggestion, I used tpm2-tools which was already included as a buildroot package (and serves me the same purpose as ibmtss). This works better and faster. Thanks for the suggestion.

Cheers,
Javier

Reply:
Nice!